nextmv/nextmv/cloud/application.py

```diff
@@ -184,9 +184,9 @@ class PollingOptions:
     """
     max_delay: float = 20
     """Maximum delay to use between polls, in seconds."""
-    max_duration: float = 300
+    max_duration: float = -1
     """Maximum duration of the polling strategy, in seconds."""
-    max_tries: int = 100
+    max_tries: int = -1
```

Comment:
Can you update the docstrings at the top? With the new defaults?

Reply:
Oh yes, sorry I forgot.